ext/opentelemetry-ext-aiohttp-client/src/opentelemetry/ext/aiohttp_client/__init__.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The opentelemetry-ext-aiohttp-client package allows tracing HTTP requests
+made by the aiohttp client library.
+
+Usage
+-----
+
+    .. code:: python
+
+        import aiohttp
+        from opentelemetry.ext.aiohttp_client import (
+            create_trace_config,
+            url_path_span_name
+        )
+        import yarl
+
+        def strip_query_params(url: yarl.URL) -> str:
+            return str(url.with_query(None))
+
+        async with aiohttp.ClientSession(trace_configs=[create_trace_config(
+                # Remove all query params from the URL attribute on the span.
+                url_filter=strip_query_params,
+                # Use the URL's path as the span name.
+                span_name=url_path_span_name
+        )]) as session:
+            async with session.get(url) as response:
+                await response.text()
+
+"""
+
+import contextlib
+import socket
+import types
+import typing
+
+import aiohttp
+
+from opentelemetry import context as context_api
+from opentelemetry import propagators, trace
+from opentelemetry.ext.aiohttp_client.version import __version__
+from opentelemetry.trace import SpanKind
+from opentelemetry.trace.status import Status, StatusCanonicalCode
+
+
+# TODO: refactor this code to some common utility
+def http_status_to_canonical_code(status: int) -> StatusCanonicalCode:
```

Comment:
It does, hence the `TODO`. Not sure where it could be shared from though. `opentelemetry-sdk` perhaps? I'm not very familiar with the OTel project structure